openapi-cli/src/test/java/io/ballerina/openapi/generators/service/ServiceContractTests.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Do we use `All rights reserved` now ?

Reply:
It should not be there. I have updated the license header, thanks for pointing it out!